tests/spec/fs.appendFile.spec.js

```diff
@@ -110,11 +110,22 @@ describe('fs.appendFile', function() {
     });
   });
 
-  it('should work when file does not exist, and create the file', function(done) {
+  it('should error when the symbolic link does not exist', function(done) {
+    var fs = util.fs();
+     
+    fs.symlink('/myfile', ' ', function(error) {  
```

Comment:
We want to avoid putting unrelated tests in a file with other functions.  Here you've got a test for `symlink` in a file for tests of `appendFile`.  Did you want to try and rewrite this in terms of something to do with `appendFile`?  If not, do you want to move this test to the `symlink` test file?